benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -24,7 +23,7 @@ def __init__(self):
         self._vm_name = ''
         self._data_dict = {}
         self._bootstorm_start_time = {}
-        self._prometheus_metrics_operation = PrometheusMetricsOperation()
+
```

Comment:
I believe there's supposed to be only one blank line between method definitions (2 lines at top level).